src/zmq/zmqnotificationinterface.cpp

```diff
@@ -100,7 +100,6 @@ bool CZMQNotificationInterface::Initialize()
 
     if (i!=notifiers.end())
     {
-        Shutdown();
```

Comment:
@laanwj assuming you delete the object.. nevertheless `Initialize()` should be all or nothing. Removing the call to `Shutdown()` leaves the notification interface partially initialised.